### DISCUSSION TITLE
beets unimported plug-in confusing upper case with lower case?

### DISCUSSION THREAD

**Discussion by mkmtrading** [beetbox/beets#5501] [Q&A]
Windows 11 
beets version 2.0.0
Python version 3.12.5

Output from beet unimported plug-in is saying that almost every track already imported is unimported,  
For some reason it isn't capitalizing the M in Z:\Music...  
IMPORTED  :  Z:\Music\Albums\10cc - The Very Best Of 10cc
ERROR:          Z:\music\Albums\10cc - The Very Best Of 10cc

**Comment by eramdam** [comment]
Having the same issue 
beets version 2.5.1
Python version 3.11.2
Debian 12.12
plugins: bandcamp, convert, discogs, duplicates, edit, embedart, export, fetchart, hook, importadded, info, inline, lastgenre, musicbrainz, unimported

Example (`0h85` instead of `0H85`)

```
$ beet unimported | sort
/media/Music/beets-flac/0h85/Sidewalk Petrichor (2018)/01 Intro (Grooove).flac
/media/Music/beets-flac/0h85/Sidewalk Petrichor (2018)/02 Who U Call.flac
/media/Music/beets-flac/0h85/Sidewalk Petrichor (2018)/03 Do This Sh!t.flac
/media/Music/beets-flac/0h85/Sidewalk Petrichor (2018)/04 Pleee Doo.flac
/media/Music/beets-flac/0h85/Sidewalk Petrichor (2018)/05 Huh.flac
/media/Music/beets-flac/0h85/Sidewalk Petrichor (2018)/06 Thot Process.flac
/media/Music/beets-flac/0h85/Sidewalk Petrichor (2018)/07 Ride 2 Nite.flac
/media/Music/beets-flac/0h85/Sidewalk Petrichor (2018)/08 Outro (Lookin').flac

$ beet info "Sidewalk Petrichor" -a
/media/Music/beets-flac/0H85/Sidewalk Petrichor (2018)
```
